src/components/views/EResources.js

```diff
@@ -227,7 +227,7 @@ const EResources = ({
                       type: e => <EResourceType resource={e} />,
                       isbn: e => getResourceIdentifier(e._object, 'isbn'),
                       eissn: e => getResourceIdentifier(e._object, 'eissn'),
-                      pissn: e => getResourceIdentifier(e._object, 'pissn'),
+                      pissn: e => getSiblingIdentifier(e._object, 'issn'),
```

Comment:
This change makes it so that the the resource's `pissn` is _never_ used. I can't find any text stating that this is the intention on the JIRA; I assumed it meant "use the sibling `issn` _if_ the resource doesn't have a `pissn`". Can you confirm how it's intended to function w/ Owen or the model with Steve?

Reply:
As I understood the implementation of **siblingIdentifiers** implies that print identifiers are no longer stored under the electronic resources **identifiers** but I can ask Owen / Steve

Reply:
Huh. In that case, can you also ask Owen if we should change _all_ the places we use `pissn` and not just this one?
<img width="488" alt="Screen Shot 2020-05-28 at 10 11 43 AM" src="https://user-images.githubusercontent.com/5324374/83152294-a711fa80-a0cb-11ea-91a7-e2e8335aff19.png">